recipes-tools/fluent-bit/files/td-agent-bit.conf.mustache

```diff
@@ -10,6 +11,13 @@
   db /tmp/fluentbit.db
   tag {{fluentbit.input_tags}}
 
+[INPUT]
+  name tail
+  path /var/log/containers/*.log
```

Comment:
I would change these to /var/log/*.log

Reply:
See https://github.com/flashbots/meta-evm/pull/50#discussion_r1867467571 for the reasoning